.github/workflows/build.yml

```diff
@@ -55,10 +55,12 @@ jobs:
              then
                 echo ::set-output name=DOCKER_IMAGE::${{ env.DOCKER_REPOSITORY }}:sha-${{ steps.sha.outputs.short }}
                 echo ::set-output name=DOCKER_EVENT::${{ env.DOCKER_REPOSITORY }}:master
+                echo ::set-output name=DOCKER_BASE::${{ env.DOCKER_REPOSITORY }}:base-master
              else
                 echo ::set-output name=DOCKER_SHA::${{ env.DOCKER_REPOSITORY }}:sha-${{ steps.sha.outputs.short }}
                 echo ::set-output name=DOCKER_IMAGE::${{ env.DOCKER_REPOSITORY }}:review-${{steps.sha.outputs.short }}
                 echo ::set-output name=DOCKER_EVENT::${{ env.DOCKER_REPOSITORY }}:PR-${{ github.event.number }}
+                echo ::set-output name=DOCKER_BASE::${{ env.DOCKER_REPOSITORY }}:base-PR-${{ github.event.number }}
```

Comment:
We usually use the branch name. Can we use it for consistency with other services?

Reply:
changed DOCKER_BASE to be the branch (for PR) or master (for pull)